### ISSUE TITLE
Support for schema lists

### ISSUE THREAD

**Issue by ajmacdonald** [cwrc/CWRC-WriterBase#151]
There's currently no specific handling of the `<list>` element in the RELAX NG specification.

**Comment by ilovan** [comment]
isn't this something that is dealt with by the css properties in the declared css, @ajmacdonald ?

**Comment by ajmacdonald** [comment]
No, this has to do with specifying a format for a value. See: https://relaxng.org/tutorial-20011203.html#IDAK0YR
